src/components/RecipeCard.js

```diff
@@ -14,7 +14,8 @@ export default function RecipeCard ({ title, subTitle, image, onClick }) {
     const classes = useStyles();
 
     return (
-        <Card className={classes.root} onClick={() => {onClick()}}>
+        <Card className={classes.root} 
+            onClick={() => {onClick({image: image, title: title})}}>
```

Comment:
I am not sure if this can work, but the `onClick={() => onClick()}` after minor changes in Example.js works too, so you don't have to change anything.